java/src/enchcracker/Enchantments.java

```diff
@@ -601,8 +608,8 @@ public static int calcEnchantmentTableLevel(Random rand, int slot, int bookshelv
 				// Get optional extra enchantments
 				while (rand.nextInt(50) <= level) {
 					// 1.14 enchantment nerf
-					level = level * 4 / 5 + 1;
-					allowedEnchantments = getHighestAllowedEnchantments(level, item, treasure);
+					//level = level * 4 / 5 + 1;
+					//allowedEnchantments = getHighestAllowedEnchantments(level, item, treasure);
```

Comment:
This should probably be removed rather than commented out. Also you haven't included all the reverts made in the other PR.

Reply:
The other stuff in that other PR was code style changes, such as inlining a function since it was called only in one place.